src/main/org/nlogo/app/ModelsLibraryDialog.java

```diff
@@ -32,11 +34,6 @@ public static String open(java.awt.Frame parent)
       me = new ModelsLibraryDialog(parent);
       me.tree.setSelectionRow(0);
     }
-    // not sure I really understand why this should be necessary
-    // (it isn't on Windows), but at least on Macs, this ensures
-    // that the tree gets the keyboard focus when the dialog opens
-    // - ST 12/17/04
-    me.setFocusable(false);
```

Comment:
@SethTisue , I see that this comment is well over ten years old, but do you remember whether it was particularly important to anyone that the file tree get keyboard focus on dialog open? It seems like the search box is a much more natural place to put the focus, but perhaps that didn't exist in early iterations of this class?